src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -19,11 +21,18 @@
 
 	public static final int SCALE = extractScale();
 
+	private static final Logger logger = LoggerFactory.getLogger( ScreenshotProvider.class );
+
 	private ScreenshotProvider() {}
 
 	public static BufferedImage shoot( final WebDriver driver ) {
-		final boolean viewportOnly = Boolean.getBoolean( VIEWPORT_ONLY_SCREENSHOT_PROPERTY );
-		return viewportOnly ? shootViewportOnly( driver ) : shootFullPage( driver );
+		try {
```

Comment:
How about a test for this? You can mock the given `WebDriver` to throw an exception an do:

```java
assertThat( ScreenshotProvider.shoot( exceptionCausingDriver ) ).isNull();
```

Reply:
Done